src/main/java/frc/team4909/robot/sensors/PhotoElectricSensors.java

```diff
@@ -0,0 +1,75 @@
+package frc.team4909.robot.sensors;
+import frc.team4909.robot.Robot;
+import edu.wpi.first.wpilibj.DigitalInput;
+public class PhotoElectricSensors{  //not sure if extends or implements
+    public static DigitalInput frontLeftSensor,     
+                              frontMiddleLeftSensor, 
+                              frontMiddleRightSensor,
+                              frontRightSensor;
+    private double velocity = 0.3;
+    boolean state = false;
+    boolean state2 = false;
+    public PhotoElectricSensors(){
+        super();
+        frontLeftSensor = new DigitalInput(0);
+        BackLeftSensor = new DigitalInput(1);
+        frontRightSensor = new DigitalInput(2);
+        BackRightSensor = new DigitalInput(3);
+    }
+
+    public void onLine(){
+        
+    if(!frontLeftSensor.get() && !frontRightSensor.get()){
+        state=true;
```

Comment:
`state` is not a variable name that should have boolean values. What does it mean for `state` to be true? The variable name should be named such that it is obvious, when you set it to a boolean value, what that means.

Reply:
Besides that, it doesn't appear that either `state` or `state2` are used in this class, so the whole `if` clause doesn't do anything...